docs/docusaurus/docs/cloud/data_assets/manage_data_assets.md

```diff
@@ -168,141 +57,49 @@ The following table lists the available Data Asset metrics.
-
-1. In GX Cloud, click **Data Assets** and then select a Data Asset in the **Data Assets** list.
-
-2. Click the **Overview** tab.
-
-    When you select a new Data Asset, schema data is automatically fetched.
-
-3. Optional. Select one of the following options:
-
-    - Click **Profile Data** if you have not previously returned all available metrics for a Data Asset.
-
-    - Click **Refresh** to refresh the Data Asset metrics.
-
-4. Click **New Expectation**.
-
-5. Select one of the following options:
-
-    - To add an Expectation to a new Expectation Suite, click the **New Suite** tab and then enter a name for the new Expectation Suite.
-
-    - To add an Expectation to an existing Expectation Suite, click the **Existing Suite** tab and then select an existing Expectation Suite.
-
-6. Select an Expectation type. See [Available Expectation types](/cloud/expectations/manage_expectations.md#available-expectation-types).
-
-7. Complete the fields in the **Create Expectation** pane.
-
-8. Click **Save** to add the Expectation, or click **Save & Add More** to add additional Expectations.
-
-
-## Add a Data Asset to an Existing Data Source
-
-Additional Data Assets can only be added to Data Sources created in GX Cloud.
-
-1. In GX Cloud, click **Data Assets** and then select **New Data Asset**.
-
-2. Click the **Existing Data Source** tab and then select a Data Source.
+## Edit a Data Asset
 
-3. Click **Add Data Asset**.
+You can only edit the settings of Data Assets created in GX Cloud.
 
-4. Select tables to import as Data Assets:
+1. In GX Cloud, click **Data Assets** and in the Data Assets list click **Edit Data Asset** for the Data Asset you want to edit.
 
-    - Check the box next to a table name to add that table as an asset.
+2. Edit the following fields:
 
-    - At least one table must be added.
+    - **Table name**: Enter the name of a new table from the Data Source to import as a Data Asset.
 
-    - To search for a specific table type the table's name in the **Search** box above the list of tables.
+    - **Data Asset name**: Enter a new name for the Data Asset. If you use the same name for multiple Data Assets, each Data Asset must be associated with a unique Data Source.
 
-    - To add all of the available tables check the box for **All Tables**.
+3. Click **Save**.
 
+## Delete a Data Asset
 
-5. Click **Add Asset**.
 
+1. In GX Cloud, click **Data Assets**.
+2. In the Data Assets list, click **Delete Data Asset** for the Data Asset you want to remove.
+3. Review the warning and click **Delete** to confirm.
 
 ## Edit Data Source settings
```

Comment:
Do you think it would make sense to add a **Manage Data Sources** section to the TOC that contains *Edit Data Source settings* and *Data Source credential management*? I'm thinking about readers that might be looking for information on how to edit or delete a Data Source, but don't find it since it's included under the Data Assets section.

Reply:
Yes, I think that's a good idea. I almost starting doing that before requesting review on this PR but dismissed it as scope creep. Since you also think it would be helpful I'll file a follow-up issue to do this in the future.